src/metrics/mongoose_metrics_hooks.erl

```diff
@@ -11,16 +11,17 @@
 -include("mongoose.hrl").
 -include("jlib.hrl").
 
--export([get_hooks/1]).
+-export([hooks/1]).
+-export([c2s_hooks/1]).
 
 %%-------------------
 %% Internal exports
 %%-------------------
 -export([sm_register_connection_hook/5,
          sm_remove_connection_hook/5,
          auth_failed/3,
-         user_send_packet/4,
-         user_receive_packet/5,
+         user_send_packet/3,
+         user_open_session/3,
```

Comment:
Remember these changes well because we will have conflicts in the future with the hook reworks 😛